airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1039,7 +1039,7 @@ def execute(self, context: Context):
             self.log.info("Total extracted rows: %s", len(rows))
 
             if self.as_dict:
-                table_data = [{k: v for k, v in row.items()} for row in rows]
+                table_data = [dict(row.items()) for row in rows]
```

Comment:
```suggestion
                table_data = [dict(row) for row in rows]
```

I wonder if this is enough.

Reply:
It is enough!
```python
>>> from google.cloud.bigquery.table import Row
>>> row = Row(('a', 'b'), {'x': 0, 'y': 1})
>>> dict(row)
{'x': 'a', 'y': 'b'}
>>> dict(row.items())
{'x': 'a', 'y': 'b'}
```